charts/shield/values.yaml

```diff
@@ -324,6 +324,8 @@ cluster:
     labels: {}
     # The annotations for the service account
     annotations: {}
+  # The default security context of the cluster shield pods
+  security_context: {}
```

Comment:
This has been named `custom_security_context` on host shield, shouldn't we strive for consistency? Or is it a different thing here?

Reply:
+1 For the sake of consistency i'll put the same naming